src/main/java/no/nav/tag/kontakt/oss/fylkesinndelingMedNavEnheter/FylkesinndelingScheduler.java

```diff
@@ -0,0 +1,58 @@
+package no.nav.tag.kontakt.oss.fylkesinndelingMedNavEnheter;
+
+import net.javacrumbs.shedlock.core.LockConfiguration;
+import net.javacrumbs.shedlock.core.LockingTaskExecutor;
+import no.nav.tag.kontakt.oss.fylkesinndelingMedNavEnheter.integrasjon.NorgService;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Component
+public class FylkesinndelingScheduler {
+    private final FylkesinndelingRepository fylkesinndelingRepository;
+    private final LockingTaskExecutor taskExecutor;
+    private final NorgService norgService;
+
+    public FylkesinndelingScheduler(FylkesinndelingRepository fylkesinndelingRepository, LockingTaskExecutor taskExecutor, NorgService norgService) {
+        this.fylkesinndelingRepository = fylkesinndelingRepository;
+        this.taskExecutor = taskExecutor;
+        this.norgService = norgService;
+    }
+
+    @Scheduled(cron = "* * * * * ?")
+    public void scheduledOppdaterInformasjonFraNorg() {
```

Comment:
Hvis jeg har skjønt det riktig vil denne kjøre hvert sekund, men låsen sørger for at den maks kjører en gang hver 24. time? Det er i grunnen en grei fremgangsmåte, men kanskje litt voldsomt at jobben kjører _hvert_ sekund når vi ikke ønsker at den skal kjøre oftere enn en gang i døgnet. Kanskje hadde det vært nok med en gang i minuttet eller halvminuttet.

Reply:
Endret til kjøring hver time